core/market/decentralized/tests/test_negotiations.rs

```diff
@@ -68,7 +66,8 @@ async fn test_exchanging_draft_proposals() -> Result<(), anyhow::Error> {
         Some(identity1.identity.to_string()),
     );
     assert_eq!(proposal1_prov.state, Some(State::Draft));
-    assert_eq!(proposal1_prov.prev_proposal_id, None);
+    // Requestor and Provider have different initial Proposals IDs
```

Comment:
This is not the initial proposal. This is the first one seen on the Provider side.
```suggestion
    // Requestor and Provider have different first Proposals IDs
```